internal/cmd/operator-sdk/pkgmantobundle/cmd.go

```diff
@@ -308,8 +308,9 @@ func getSDKStamps(bundle *apimanifests.Bundle) (map[string]string, error) {
 	return sdkLabels, nil
 }
 
-// getChannelsByCSV creates a list for channels for the currentCSV,
-func getChannelsByCSV(bundle *apimanifests.Bundle, channelsByCSV map[string][]string) (channels string) {
+// getChannelsByCSV creates a list for channels for the currentCSV. For other versions of manifests which
+// are not present in the manifest, the defaultChannel is added.
+func getChannelsByCSV(bundle *apimanifests.Bundle, channelsByCSV map[string][]string, defaultChannel string) (channels string) {
 	// Find channels matching the CSV names
 	var channelNames []string
 	for csv, ch := range channelsByCSV {
```

Comment:
Side note: you can do 
```suggestion
	channelNames := channelsByCSV[bundle.CSV.GetName()]
```
instead of a loop